Map/RoomBase.gd

```diff
@@ -7,13 +7,10 @@ class_name RoomBase
 @export var light_level: Enums.LightLevel:
 	set(value):
 		light_level = value
-		on_light_level_changed.emit(light_level)
```

Comment:
why is it not needed anymore ?

Reply:
This was to handle updating the lightoverlay whenever light changes, but since light changes based on movement and putting a torch down which is done in MapUI, we don't need a signal to pick up when light is changed.

Reply:
I see